README.md

```diff
@@ -4,56 +4,53 @@
+**:loudspeaker: Important:** The Web3 Foundation _General Grants_ Program focuses on technology grants which teams wish to keep **private**, need to be **paid out in fiat** or are **larger than $30,000**. We ask teams to first consider applying via the [_Open Grants_ Program](https://github.com/w3f/Open-Grants-Program), which supports grants of up to $30,000 and has **faster processing times**. If your project has a scope larger than that, we encourage you to split your work into multiple smaller stages of $30,000 or less.
 
 ---
 
 - [:wave: Introduction](#wave-introduction)
-  - [About](#about)
-  - [Licensing](#licensing)
-  - [Documentation](#documentation)
+  - [Guidelines](#guidelines)
 - [:pencil: Process](#pencil-process)
-- [:mailbox_with_mail: RFP Suggestion](#mailbox_with_mail-rfp-suggestion)
+- [:mailbox_with_mail: RFP Suggestions](#mailbox_with_mail-rfp-suggestions)
 - [:bulb: Help](#bulb-help)
   - [Additional information](#additional-information)
   - [Real-time conversation](#real-time-conversation)
 - [:rocket: Alternative Funding Sources](#rocket-alternative-funding-sources)
   - [Treasury](#treasury)
   - [Open Grants Program](#open-grants-program)
   - [Other Grant Programs](#other-grant-programs)
-- [:information_source: License](#information_source-license)
 
 ## :wave: Introduction
 
-As part of our commitment to promoting the Web3 ecosystem, we offer a comprehensive grants program focused on funding software development and research efforts related to **Polkadot, Kusama and Substrate**. For more information about the Web3 Foundation please visit the [About page](https://web3.foundation/about/) of our website.
+As part of our commitment to promoting the Web3 ecosystem, we offer comprehensive grants programs focused on funding software development and research efforts related to **Polkadot, Kusama and Substrate**. For more information about the Web3 Foundation please visit the [About page](https://web3.foundation/about/) of our website.
 
 **Only apply via the General Grants Program if:**
 
-- You seek funding of $30,000+ and your work cannot be split into smaller milestones
-- You want your team/project information to remain private
-- You can only accept fiat payments
+- you seek funding of $30,000+ and your work cannot be split into smaller milestones,
+- you want your team/project information to remain private, or
+- you can only accept fiat payments.
 
 Otherwise, you should apply to the [Open Grants Program](https://github.com/w3f/Open-Grants-Program).
 
-Furthermore, teams shouldn’t seek to cover 100% of their early-stage funding via W3F Grants alone.
+Any project requesting **$30k or less** might receive fast approval. Projects between **$30k and $100k** will likely face longer waiting times.
```

Comment:
Is this true for general grants? Or does this refer to the open grants program?

Reply:
I think this was intentionally here, since it is true that we look at grants with more scrutiny if the amount is larger. Also Open Grants are plainly not allowed to surpass $30k unless they are a follow-up